cmake/Modules/Generate.cmake

```diff
@@ -15,6 +15,14 @@ function(IDLC_GENERATE)
   cmake_parse_arguments(
     IDLC "" "${one_value_keywords}" "${multi_value_keywords}" "" ${ARGN})
 
+  if (CMAKE_CROSSCOMPILING)
+    find_program(IDLC_EXECUTABLE idlc REQUIRED NO_CMAKE_FIND_ROOT_PATH)
+    set(IDLC_COMMAND_DEPENDS "")
```

Comment:
The `IDLC_EXECUTABLE` might clash with a variable defined somewhere else(?) (not sure). I think I'd just use `_command` and `_depends`. It's common for local variables in a function to start with an underscore in CMake to indicate they're local etc. Might even be that there's a couple of additional properties (hidden?), but I'm not a 100% sure about that.